docs/integrations/fastapi.md

```diff
@@ -97,10 +110,64 @@ app.include_router(graphql_app, prefix="/graphql")
 ```
 
 Here we are returning a custom context dictionary that contains one extra item
-called "custom_value", which is injected from `custom_context_dependency`.
+called "custom*value", which is injected from `custom_context_dependency`. This
+value exists alongside `request`, `response`, and `background_tasks` in the
+`info.context` \_dictionary* and so it requires `['request']` indexing.
+
+Then we use the context in a resolver.
+The resolver will return "Hello John" in this case.
+
+For class-based custom contexts, an example might look like the following.
+
+```python
+import strawberry
+
+from fastapi import FastAPI, Depends, Request, WebSocket, BackgroundTasks
+from strawberry.types import Info
+from strawberry.fastapi import BaseContext, GraphQLRouter
+
+
+class CustomContext(BaseContext):
+    def __init__(self, greeting: str, name: str):
+        self.greeting = greeting
+        self.name = name
+
+
+def custom_context_dependency() -> CustomContext:
+    return CustomContext(greeting = "you rock!", name = "John")
```

Comment:
```suggestion
    return CustomContext(greeting="you rock!", name="John")
```

Reply:
To be clear here, is this a style convention that Strawberry adheres too? Is it also documented in a PEP somewhere? I ask because I prefer whitespace in those instances and would like to know if I'm wrong!

Reply:
@tsmith023 We generally follow black formatting in our codebase and in the docs.

Reply:
This is also a PEP-8 recommendation:
https://www.python.org/dev/peps/pep-0008/#other-recommendations

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:
>
> ```python
> # Correct:
> def complex(real, imag=0.0):
>     return magic(r=real, i=imag)
> ```
> ```python
> # Wrong:
> def complex(real, imag = 0.0):
>     return magic(r = real, i = imag)